### PR TITLE
Creating Trapheus.yml

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: _data/projects/Trapheus.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+tags:
+  - rds
+  - step-functions
+  - python
+  - aws-lambda
+  - aws-lambda-layers
+  - database-restore
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/username/project/labels/up%20for%20grabs
```
Comment (shiftkey) [inline _data/projects/Trapheus.yml]: This needs to be a URL that points to the list of issues labelled with `up for grabs`. If you agree that the label should be `good first issue` then we can update this too:

```suggestion
  link: https://github.com/intuit/Trapheus/labels/good%20first%20issue
```

File: _data/projects/Trapheus.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+site: https://github.com/intuit/Trapheus
+tags:
+  - rds
+  - step-functions
+  - python
+  - aws-lambda
+  - aws-lambda-layers
+  - database-restore
+upforgrabs:
+  name: up for grabs
```
Comment (shiftkey) [inline _data/projects/Trapheus.yml]: I don't see this label being used in your project - should this be changed to `good first issue`?

```suggestion
  name: good first issue
```

File: _data/projects/Trapheus.yml (hunk 1 above)
Comment (ghost) [reply _data/projects/Trapheus.yml]: @shiftkey  Made the required changes

File: _data/projects/Trapheus.yml (hunk 2 above)
Comment (ghost) [reply _data/projects/Trapheus.yml]: @shiftkey  Changes made.

**Review by shiftkey — APPROVED** [review]
Thanks for the contribution @stationeros!

**Pull request merged**